draft-packages/title-block-zen/KaizenDraft/TitleBlockZen/TitleBlockZen.tsx

```diff
@@ -29,8 +29,6 @@ type DistributiveOmit<T, K extends keyof any> = T extends any
 
 export const NON_REVERSED_VARIANTS = ["education", "admin"]
 
-type AvatarProps = Omit<DraftAvatarProps, "size">
```

Comment:
Curious why this was removed, 'cause it looks like it will still be forced to be `medium` size when passed into the title block

Reply:
I removed this because its was causing a TS compile error when it came up against updated Avatar Props. I think issue lies in trying to omit size from the union type of GenericAvatarProps and CompanyAvatarProps. I could potentially bring this back if you see it causing an issue by importing GenericAvatarProps over Avatar props but given size is set of ln 238 to medium as the default I'm not sure if that is necessary? Would be keen to get your thoughts

Reply:
I think it's to stop people passing in a size, because regardless of what is passed in, it will always become medium (see https://github.com/cultureamp/kaizen-design-system/blob/master/draft-packages/title-block-zen/KaizenDraft/TitleBlockZen/TitleBlockZen.tsx#L240).

If you can make it work, I think it does make sense to keep it.

Reply:
Found a solution to get the best of both, I've pull in the individual interfaces and omitted the size prop
```
type AvatarProps =
  | Omit<GenericAvatarProps, "size">
  | Omit<CompanyAvatarProps, "size">
```